components/tyk-gateway/templates/deployment-gw-repset.yaml

```diff
@@ -96,6 +96,26 @@ spec:
             value: "{{ .Values.gateway.control.containerPort }}"
           {{- end }}
 
+          # Redis TLS configurations
+          - name: TYK_GW_STORAGE_USESSL
+            value: "{{ .Values.global.redis.useSSL }}"
+          - name: TYK_GW_STORAGE_SSLINSECURESKIPVERIFY
+            value: "{{ .Values.global.redis.sslInsecureSkipVerify }}"
+          - name: TYK_GW_STORAGE_SSLCAFILE
+            value: "{{ .Values.global.redis.sslCAFile }}"
+          - name: TYK_GW_STORAGE_SSLCERTFILE
+            value: "{{ .Values.global.redis.sslCertFile }}"
+          - name: TYK_GW_STORAGE_SSLKEYFILE
+            value: "{{ .Values.global.redis.sslKeyFile }}"
+          - name: TYK_GW_STORAGE_SSLMAXVERSION
+            value: "{{ .Values.global.redis.sslMaxVersion }}"
+          - name: TYK_GW_STORAGE_SSLMINVERSION
+            value: "{{ .Values.global.redis.sslMinVersion }}" 
+          - name: TYK_GW_STORAGE_REDISUSESSL
```

Comment:
Gateway doesn't have `TYK_GW_STORAGE_REDISUSESSL` and `TYK_GW_STORAGE_REDISINSECURESKIPVERIFY`environment variables.